static/css/main.css

```diff
@@ -26,7 +26,6 @@
   box-sizing: border-box;
 }
 a,
-a:visited,
```

Comment:
Thanks! Just a question. Did you edit the `dev/less/*.less` files to make these changes? If so, commit the `.less` files as well.

There's instructions in the [`README.md`](https://github.com/nishanths/cocoa-hugo-theme/tree/master/dev), in case it helps.

Reply:
I'll updated the *.less files. However, since the CSS files are auto-generated, they should be excluded from the repo. In another word: DRY.